### PR TITLE
Day4_CreatingApp_Assignment

### PR DESCRIPTION
**Pull request by sow2245**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,109 +1,120 @@
...
-//Composition of components
-
-//TitleComponent is called as functional component    
-const TitleComponent = () =>
+import CardComponent from "./CardComponent.js";
+import sowmyaImg from "./Img/Sowmya.png"
+
+const title = "Strom Troopers";
+
+const data=[
```
Comment (vickydarlinn) [inline src/App.js:8]: you can make a separate file for db.

File: src/CardComponent.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const CardComponent=(props)=>{
+    debugger;
+    return (
+        <div id="card" className="cardClass">
+            <img src={props.teamMembers.img}/>
```
Comment (vickydarlinn) [inline src/CardComponent.js]: this is also good, but you can destruct them at the top,It will make your code ease.

File: src/App.js [hunk 3]
```diff
@@ -1,109 +1,120 @@
...
-passingAttrRoot.render(passingAttrHeading);
-
-//Composition of components
-
-//TitleComponent is called as functional component    
-const TitleComponent = () =>
+import CardComponent from "./CardComponent.js";
+import sowmyaImg from "./Img/Sowmya.png"
+
+const title = "Strom Troopers";
```
Comment (vickydarlinn) [inline src/App.js:6]: make a separate file for constants and save into them.

File: src/App.js (hunk 1 above)
Comment (sow2245) [reply src/App.js:8]: moving into separate file is in next assignment

File: src/App.js (hunk 3 above)
Comment (sow2245) [reply src/App.js:6]: moving into separate file is in next assignment

File: src/CardComponent.js (hunk 2 above)
Comment (sow2245) [reply src/CardComponent.js]: Implemented as per your comments

**Comment by ShivamPandey001** [comment]
Hi @sow2245 , The changes looks good to me

**Pull request merged**